frontend/src/hooks/useReceivingCaptions.tsx

```diff
@@ -0,0 +1,48 @@
+import { Subscriber } from '@vonage/client-sdk-video';
+import { useState, useEffect, useMemo } from 'react';
+
+export type CaptionsType = {
+  captionText: string;
```

Comment:
Pedantic: maybe `caption` is sufficient, since that's a "caption text".

I'll shut up now :-)